Xwt.XamMac/Xwt.Mac/Util.cs

```diff
@@ -240,6 +240,21 @@ public static NSBitmapImageFileType ToMacFileType (this ImageFileType type)
 			}
 		}
 
+		public static bool TriggerContextMenu (this NSEvent theEvent)
+		{
+			if (theEvent.ButtonNumber == 1 &&
+					(NSEvent.CurrentPressedMouseButtons & 1 | NSEvent.CurrentPressedMouseButtons & 4) == 0) {
+				return true;
+			}
+
+			if (theEvent.ButtonNumber == 0 && Keyboard.CurrentModifiers == ModifierKeys.Control &&
```

Comment:
Instead of `Keyboard.CurrentModifiers == ModifierKeys.Control` we should probably use `(theEvent.ModifierFlags & NSEventModifierMask.ControlKeyMask) != 0`